srcpkgs/coturn/template

```diff
@@ -1,27 +1,23 @@
 # Template file for 'coturn'
 pkgname=coturn
-version=4.5.1.3
-revision=4
+version=4.6.0
+revision=1
 build_style=configure
 configure_args="
  --prefix=/usr
  --manprefix=/usr/share
  --sysconfdir=/etc"
 conf_files="/etc/turnserver.conf"
+hostmakedepends="pkg-config"
 makedepends="openssl-devel libevent-devel hiredis-devel sqlite-devel
  postgresql-libs-devel libmariadbclient-devel"
```

Comment:
it seems to not be finding libmariadbclient:
```
MySQL not found. Building without MySQL support.
```

Reply:
Huh, the pkg-config files are in the mariadb package...

```
$ xlocate /usr/lib/pkgconfig | rg maria
mariadb-10.5.10_3       /usr/lib/pkgconfig/libmariadb.pc
mariadb-10.5.10_3       /usr/lib/pkgconfig/mariadb.pc
```